seed_isort_config.py

```diff
@@ -18,10 +18,10 @@
     '.editorconfig', '.isort.cfg', 'setup.cfg', 'tox.ini', 'pyproject.toml',
 )
 THIRD_PARTY_RE = re.compile(
-    r'^known_third_party([ \t]*)=([ \t]*)(?:.*?)?(\r?)$', re.M,
+    r'^([ \t]*)known_third_party([ \t]*)=([ \t]*)(?:.*?)?(\r?)$', re.M,
```

Comment:
does `\s` work?

Reply:
I don't think `\s` is a good idea because it also matches newline characters doesn't it?

Reply:
Yes `\s` does work.

But my concern is still there. If you want I will change it.

Reply:
seems simpler if it works, doesn't really matter though I guess

Reply:
Then I will let it stay as `[ \t]` to be explicit.